cloud/google/gce.py

```diff
@@ -87,10 +87,18 @@
       - your GCE project ID
     required: false
     default: null
-  name:
+  base_name:
```

Comment:
@supertom Okay that makes sense. I'll push the changes when I get a chance. Thanks!